webcompat/static/css/development/components/markdown.css

```diff
@@ -21,9 +21,8 @@
   /* all element */
   .wc-Markdown * {
     font-size: var(--Markdown-fontSize) !important;
-    word-wrap: break-word;
-    white-space: normal;
```

Comment:
These were introduced to fix the code overflow in 
https://github.com/webcompat/webcompat.com/commit/1b6b80d92c30d14b616f9dfc5222b567f6185c43